phpstan.neon.dist

```diff
@@ -1,5 +1,5 @@
 parameters:
-    #phpVersion: 50400 # Needs to be 70100 or higher... sigh...
+    phpVersion: 70100 # Needs to be 70100 or higher... sigh...
```

Comment:
Is the comment still needed here?

Reply:
As the code base has a minimum PHP version of 5.4, I think it still has value as it explains why the `phpVersion` is not set to `50400`.